developer-workflow/c-api.rst

```diff
@@ -88,6 +95,40 @@ Guidelines for expanding/changing the public API:
   fields, arguments and return values are well defined.
 
 
+C API Tests
+-----------
+
+Tests for the public C API live in the ``_testcapi`` module.
+Functions named ``test_*`` are used as tests directly.
+Tests that need Python code (or are just easier to partially write in Python)
+live in ``Lib/test``, mainly in :file:`Lib/test/test_capi.py`.
+
+Due to its size, the ``_testcapi`` module is defined in several source
+files.
+To add a new set of tests (or extract a set out of the monolithic
+:file:`Modules/_testcapimodule.c`):
+
+- Create a C named ``Modules/_testcapi/yourfeature.c``
+
+- The file should define a module as usual, except:
+
+  - Instead of ``<Python.h>``, include ``"parts.h"``.
+  - Instead of ``PyInit_modname``, define ``_PyTestCapi_Init_yourfeature``
+    that *takes* the ``_testcapi`` module and adds functions/classes to it.
+    (Use ``PyModule_AddFunctions`` to add functions).
```

Comment:
```suggestion
  - Instead of ``PyInit_modname``, define a ``_PyTestCapi_Init_yourfeature``
    that *takes* the ``_testcapi`` module and adds functions/classes to it
    (use ``PyModule_AddFunctions`` to add functions).
```

Reply:
Thanks!
I'd rather keep the note as a separate sentence, but even then the punctuation needs a fix.

Reply:
```suggestion
  - Instead of ``PyInit_modname``, define a ``_PyTestCapi_Init_yourfeature``
    function that *takes* the ``_testcapi`` module and adds functions/classes
    to it. (You can use ``PyModule_AddFunctions`` to add functions.)
```